MCP_Server/connection.py

```
@@ -0,0 +1,215 @@
+
+        self._connected = False
+        max_attempts = 3
+        for attempt in range(1, max_attempts + 1):
+            logger.info(f"Connecting to Ableton (attempt {attempt}/{max_attempts})...")
+            if await self.connect():
+                # Validate with a quick command
+                try:
+                    await self._send_and_receive("get_session_info")
+                    logger.info("Connection validated successfully")
+                    return
+                except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)